tests/foreman/cli/test_oscap_tailoringfiles.py

```diff
@@ -26,28 +26,29 @@
 from robottelo.config import settings
 from robottelo.constants import SNIPPET_DATA_FILE
 from robottelo.datafactory import invalid_names_list
+from robottelo.datafactory import parametrized
 from robottelo.datafactory import valid_data_list
 from robottelo.decorators import tier1
 from robottelo.decorators import tier2
 from robottelo.decorators import tier4
 from robottelo.decorators import upgrade
 from robottelo.helpers import file_downloader
 from robottelo.helpers import get_data_file
-from robottelo.test import CLITestCase
 
 
-class TailoringFilesTestCase(CLITestCase):
+class TestTailoringFiles:
     """Implements Tailoring Files tests in CLI."""
 
-    @classmethod
-    def setUpClass(cls):
-        super(TailoringFilesTestCase, cls).setUpClass()
-        cls.tailoring_file_path = file_downloader(
+    @pytest.fixture(scope="class")
+    def tailoring_file_path(self):
+        tailoring_file_path = file_downloader(
             file_url=settings.oscap.tailoring_path, hostname=settings.server.hostname
         )[0]
+        return tailoring_file_path
```

Comment:
is this fixture something that could be useful outside of this file?

Reply:
yes, thanks for noticing it. I have added aaa815b813f2bcc0a9efc79f7b89badf86ce204b commit for it.

Reply:
Thank you for externalizing it, Jameer!